changelog.md

```diff
@@ -1,3 +1,15 @@
+3.3.1
+========
+- Updated to Microsoft.Identity.Model.* 8.2.0
+
+### Supportability
+- Added JSON schema support for Microsoft.Identity.Web configuration. This allows for schema validation in the `appsettings.json`, improving configuration accuracy and developer experience. To use it, add the following at the top of your appsettings.json:
+`"$schema": "https://raw.githubusercontent.com/AzureAD/microsoft-identity-web/refs/heads/jmprieur/jsonschema/JsonSchemas/microsoft-identity-web.json"`
```

Comment:
This should reference the file in the main branch.

Reply:
true, good point.